src/course_bot/quiz.clj

```diff
@@ -296,7 +296,9 @@
                (let [{quiz-key :key quiz :quiz} (current-quiz! tx conf)
                      question-index (count results)
                      next-question-index (+ 1 question-index)
-                     new-results (concat results (list text))]
+                     new-results (concat results (list text))
+                     {student-name :name group :group} (codax/get-at tx [id])
```

Comment:
Avoid db access onveach answer

Reply:
moved the read from db to the end of the quiz